Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -358,41 +357,37 @@ namespace AzToolsFramework::ViewportUi::Internal
 
     void ViewportUiDisplay::InitializeUiOverlay()
     {
-        AZStd::string styleSheet;
-
         m_uiMainWindow.setObjectName(QString("ViewportUiWindow"));
         ConfigureWindowForViewportUi(&m_uiMainWindow);
-        m_uiMainWindow.setVisible(false);
+        m_uiMainWindow.hide();
 
         m_uiOverlay.setObjectName(QString("ViewportUiOverlay"));
         m_uiMainWindow.setCentralWidget(&m_uiOverlay);
-        m_uiOverlay.setVisible(false);
+        m_uiOverlay.hide();
 
         // remove any spacing and margins from the UI Overlay Layout
         m_fullScreenLayout.setSpacing(0);
         m_fullScreenLayout.setContentsMargins(0, 0, 0, 0);
         m_fullScreenLayout.addLayout(&m_uiOverlayLayout, 0, 0, 1, 1);
 
         // format the label which will appear on top of the highlight border
-        styleSheet = AZStd::string::format("background-color: %s; border: none;", HighlightBorderColor);
-        m_viewportBorderText.setStyleSheet(styleSheet.c_str());
+        const AZStd::string borderStyleSheet = AZStd::string::format("background-color: %s; border: none;", HighlightBorderColor);
+        m_viewportBorderText.setStyleSheet(borderStyleSheet.c_str());
```

Comment:
Why make a string using `AZStd::string` only to have to copy it to a `QString`? This would be better written as:
```suggestion
        const QString borderStyleSheet = QString("background-color: %1; border: none;").arg(HighlightBorderColor);
        m_viewportBorderText.setStyleSheet(borderStyleSheet);
```

I'd even advocate for not naming that variable, since making it `const` prevents it from being moved from:
```suggestion
        m_viewportBorderText.setStyleSheet(QString{"background-color: %s; border: none"}.arg(HighlightBorderColor));
```

Reply:
Sure that makes sense (not sure how to cleanly accept the suggestion without breaking DCO but I can make this changes (it's worth mentioning this does only happen once at start-up, so it's not really a performance concern 😉 but I agree it doesn't hurt following best practices wherever possible)

Reply:
I updated this in a few more spots too :+1: